src/components/MainNavigation/MainNavigation.stories.js

```diff
@@ -128,7 +128,7 @@ export const Regular = (args) => (
           <MainNavigationItem
             className="wfp--main-navigation__user"
             subNavigation={
-              <SubNavigation>
+              <SubNavigation style={{width: '100px'}}>
```

Comment:
Is this only needed for the documentation or also other solutions?

Reply:
sorry, was actually testing something on that. Dmytro would like to reduce the width of the subnavigation.